src/content/infanterie/mg-schuetze.svx

```diff
@@ -1,41 +1,57 @@
 ---
 title: MG-Schütze
-date: '2023-01-31'
+date: '2023-06-26'
 nav_index: 3
 ---
 
 <script>
     import Tooltip from "$lib/wikitools/Tooltip.svelte";
+    import Slideshow from '$lib/slideshow/Slideshow.svelte';
+
+    // machine gun images
+    // TODO replace images with in-game ones
+    import m249 from '$lib/images/mg-schuetze/m249_REPLACEME.jpg';
+    import mg4 from '$lib/images/mg-schuetze/mg4_REPLACEME.jpg';
 </script>
 
 ## Rollenprofil
 
-Der <Tooltip text="Maschinengewehr">MG</Tooltip>-Schütze ist eine der grundlegenden Rollen der Infanterie. Die Rolle des MG-Schützen erfordert eine Fortbildung vom Schützen und ist für Anfänger sehr gut geeignet. MG-Schützen sind ein wichtiger Bestandteil von Trupps und Gruppen der Infanterie. In Gruppen sind sie grundsätzlich in Deckungstrupps anzutreffen.
+Der Maschinengewehr-Schütze ist eine der grundlegenden Rollen der Infanterie. Die Rolle des MG-Schützen erfordert eine Fortbildung vom Schützen und ist für Anfänger sehr gut geeignet. MG-Schützen sind ein wichtiger Bestandteil von Trupps und Gruppen der Infanterie. In Gruppen sind sie üblicherweise in Deckungstrupps anzutreffen.
 
 ## Aufgaben
 
-MG-Schützen haben ein relativ simples Aufgabenprofil, das allerdings von großer Bedeutung für den infanteristischen Kampf ist. Zusätzliche Aufgaben der MG-Schützen neben denen der Schützen sind unter anderem:
+MG-Schützen haben ein relativ überschaubares Aufgabenprofil, das allerdings von großer Bedeutung für den infanteristischen Kampf ist. Zusätzliche Aufgaben der MG-Schützen neben denen der Schützen sind unter anderem:
 
-- Feuerüberlegenheit herstellen
-- Niederhaltungs-/Sperrfeuer geben
+- Herstellung der Feuerüberlegenheit
+- Niederhaltung feindlicher Infanterie
+- Decken eigener Kräfte, besonders bei abwechselnder Bewegung im Feld
 
 ## Ausrüstung
 
-Neben der Grundausrüstung eines Schützen verfügen MG-Schützen über ein Maschinengewehr und die dazugehörige Munition in einem großen Rucksack. Maschinengewehre gibt es in vielen verschiedenen Variationen, darunter variiert oftmals das Gewicht der Waffe, die verschossene Munitionsart und die Magazingröße. Grundsätzlich unterscheiden wir zwischen _leichten_ und _schweren_ Maschinengewehren.
+Neben der Grundausrüstung eines Schützen verfügen MG-Schützen über ein Maschinengewehr und die dazugehörige Munition in einem großen Rucksack. Maschinengewehre gibt es in verschiedenen Variationen, darunter variiert oftmals das Gewicht der Waffe, die verschossene Munitionsart und die Magazingröße. Grundsätzlich unterscheiden wir zwischen _leichten_, _mittleren_ und _schweren_ Maschinengewehren, die in den folgenden Absätzen erläutert werden:
```

Comment:
```suggestion
Neben der Grundausrüstung eines Schützen verfügen MG-Schützen über ein Maschinengewehr und die dazugehörige Munition in einem großen Rucksack. Maschinengewehre gibt es in verschiedenen Bauweisen mit unterschiedlichen Gewichten, verschossenen Munitionsarten und Magazingrößen. Grundsätzlich unterscheidet man _leichte_, _mittlere_ und _schwere_ Maschinengewehre, die in den folgenden Absätzen erläutert werden:
```

Reply:
"unterscheiden wir" instead of "unterscheidet man" was done on purpose, since some arma units only differenciate between light and heavy machine guns, therefore basically merging light and medium machine guns into one

Reply:
I see, thanks for clarifying.

```suggestion
Neben der Grundausrüstung eines Schützen verfügen MG-Schützen über ein Maschinengewehr und die dazugehörige Munition in einem großen Rucksack. Maschinengewehre gibt es in verschiedenen Bauweisen mit unterschiedlichen Gewichten, verschossenen Munitionsarten und Magazingrößen. Grundsätzlich unterscheiden wir _leichte_, _mittlere_ und _schwere_ Maschinengewehre, die in den folgenden Absätzen erläutert werden.
```